crates/uv/src/cli.rs

```diff
@@ -846,6 +846,27 @@ pub(crate) struct PipSyncArgs {
     #[arg(long, short = 'C', alias = "config-settings")]
     pub(crate) config_setting: Option<Vec<ConfigSettingEntry>>,
 
+    /// The minimum Python version that should be supported by the requirements (e.g.,
+    /// `3.7` or `3.7.9`).
+    ///
+    /// If a patch version is omitted, the most recent known patch version for that minor version
+    /// is assumed. For example, `3.7` is mapped to `3.7.17`.
+    #[arg(long)]
+    pub(crate) python_version: Option<PythonVersion>,
+
+    /// The platform for which requirements should be installed.
+    ///
+    /// Represented as a "target triple", a string that describes the target platform in terms of
+    /// its CPU, vendor, and operating system name, like `x86_64-unknown-linux-gnu` or
+    /// `aaarch64-apple-darwin`.
+    ///
+    /// WARNING: When specified, uv will select wheels that are compatible with the target platform.
+    /// The resulting environment may not be fully compatible with the current platform. Further,
+    /// distributions that are built from source may ultimately be incompatible with the target
+    /// platform. This option is intended for cross-compilation and other advanced use cases.
```

Comment:
This last sentence is confusing given the preceding paragraph, perhaps you meant

```suggestion
    /// platform. This option is not intended for cross-compilation and other advanced use cases.
```
?

Reply:
I think "cross-compilation" is the wrong term. It _is_ intended for advanced use-cases, but "cross-compilation" emphasizes building from source in the wrong way.

Reply:
Ah, okay. Yeah, I was definitely also mislead by the preceding sentence that specifically talks about building from source.